.github/workflows/test_published_images.yml

```diff
@@ -178,13 +183,13 @@ jobs:
           if [[ "${{ matrix.distribution-type }}" == "ee" && -z "${{ matrix.variant }}" ]]; then
             # NLC repo only populated for absolute versions - not "latest", "latest-lts" etc tags
             # Identify absolute version based on earlier parsing of version number
-            if [[ -n "${{ steps.version.outputs.major }}" ]]; then
+            if [[ -n "${{ steps.image-version.outputs.major }}" ]]; then
```

Comment:
nit - could make `steps.image-version.outputs.major` as local var so its shorter

Reply:
ignore. my bad. sorry
Thought it was used couple of times but next line uses different var `steps.expected-hz-version.outputs.major`
kind of half read `version.outputs.major` and thought it was the same